File: 2.If Else Statements/solution.py
class Solution:
    def compareNM(self, n : int, m : int) -> str:
        if n<m:
            return "lesser"
        elif n>m:
            return "greater"
        else:
            return "equal"

if __name__=="__main__":
    t = int(input())
    for _ in range(t):
        
        n,m=map(int,input().split())
        
        obj = Solution()
        res = obj.compareNM(n, m)
        
        print(res)

File: 2.If Else Statements/solution.js
let readline = require("readline");

let count = 0;
let testcasesCount = null;

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

const ifElseStatment = function (inputs) {
  const [n, m] = [...inputs];
  if (n < m) {
    return "lesser";
  } else if (n > m) {
    return "greater";
  } else {
    return "equal";
  }
};

rl.on("line", (data) => {
  if (testcasesCount === null) {
    testcasesCount = parseInt(data);
  } else {
    count++;
    console.log(ifElseStatment(data.split(" ")));
    if (count === testcasesCount) {
      rl.close();
    }
  }
});
